src/components/ec-button-group/ec-button-group.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div class="ec-btn-group">
+    <button
+      v-for="(button, index) in items"
+      :key="index"
+      class="ec-btn ec-btn--md ec-btn--primary ec-btn-group__btn"
+      :class="{
+        'ec-btn--outline': value !== button.value
+      }"
+      :disabled="button.disabled"
+      @click="$emit('change', button.value)"
+    >
+      {{ button.text }}
+    </button>
+  </div>
+</template>
+
+<script>
+import { arrayOfObjectsContainsKey } from '../../utils/validators';
+
+export default {
+  name: 'EcButtonGroup',
+  model: {
+    prop: 'value',
+    event: 'change',
+  },
+  props: {
+    value: [Number, String, Boolean],
```

Comment:
I wouldn't limit the type of the value, what if I want to pass a object as a value?